code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -513,7 +513,12 @@
 	caliber = "laser"
 	max_ammo = 20
 	multi_sprite_step = 5
+	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/ammo_box/magazine/laser/ert //Used by red ERT. Keeps the size for them
+	name = "compact laser carbine projector magazine"
+	desc = "By use of bluespace technology, one can store many more lasers inside this magazine than otherwise would be reasonable."
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
Could you add an empty line between lines 521 and 522 to make it easier to read? Also, maybe these magazines should be EMP proof, ERT are usually pretty resistant to EMPs

Reply:
Sure. Can make them emp proof as well actually.

Reply:
Done